app/lib/actions.ts

```diff
@@ -0,0 +1,35 @@
+import { Response } from "@/app/lib/definitions";
+
+const PUBLIC_BASE_URL =
+  process.env.PUBLIC_BASE_URL ||
+  "https://f442285821a31af458af8b09d237e087.serveo.net";
+
+const CREDENTIAL_DEFINITION_ID =
+  process.env.CREDENTIAL_DEFINITION_ID ||
+  "did:web:chatbot-demo.dev.2060.io?service=anoncreds&relativeRef=/credDef/HngJhYMeTLTZNa5nJxDybmXDsV8J7G1fz2JFSs3jcouT";
+
+export async function getQR(socketConnectionId: string): Promise<Response> {
+  try {
+    const url =
+      "https://a.chatbot-demo.dev.2060.io/v1/invitation/presentation-request";
```

Comment:
Create a SERVICE_AGENT_ADMIN_BASE_URL for this in order to not be tied to this external Service Agent. It can default to `https://a.chatbot-demo.dev.2060.io` for the moment, until we set up an internal one.